composer.json

```diff
@@ -9,8 +9,7 @@
     "ext-dom": "*",
     "ext-json": "*",
     "ext-libxml": "*",
-    "ext-mbstring": "*",
-    "civicrm/composer-downloads-plugin": "^2.1"
```

Comment:
FWIW we'll need to keep that one for #4513

Reply:
Fixed.